snapshotter/cmd/devmapper/Makefile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+all: devmapper
+
+devmapper: *.go
+	go build -o devmapper_snapshotter -tags libdm_dlsym_deferred_remove
```

Comment:
If the make target (devmapper) doesn't match the name of a generate file, then this should be a .PHONY target. Or rename the target to "devmapper_snapshotter", which I think is more appropriate in this case.

Reply:
Will address